### PR TITLE
Release Testnet and SDK v0.13.1

### PR DESCRIPTION
**Pull request by ma2bd**

### CODE REVIEW

File: src/developers/sdk/deploy.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ linera publish-and-create \
   --json-argument "42"
 ```
 
-## Devnet
+## Devnets and Testnets
 
-To configure the wallet for the devnet while creating a new microchain, the
-following command can be used:
+To configure the wallet for the current testnet while creating a new microchain,
```
Comment (christos-h) [inline src/developers/sdk/deploy.md:28]: nit: inconsistent capitalisation of "Testnet".

File: src/developers/sdk/deploy.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 The first step to deploy your application is to configure a wallet. This will
 determine where the application will be deployed: either to a local net or to
-the devnet.
+the public deployment (i.e. a devnet or a testnet).
```
Comment (christos-h) [inline src/developers/sdk/deploy.md:5]: nit: inconsistent capitalisation of "Testnet".

**Review by christos-h — APPROVED** [review]

File: src/developers/getting_started/hello_linera.md [hunk 3]
```diff
@@ -1,31 +1,36 @@
...
+The Linera Testnet is a deployment of the Linera protocol useful for developers
+and able to host applications.
 
-To interact with the Devnet, some tokens are needed. A Faucet service is
+> The current Testnet (codename "Archimedes") is the first deployment of Linera
+> run in partnership with external validators. While it should be considered
+> stable, it will be replaced by a new Testnet when needed. The next Linera
+> Testnet will be restarted from a clean slate and a new genesis block.
+
+To interact with the Testnet, some tokens are needed. A Faucet service is
```
Comment (afck) [inline src/developers/getting_started/hello_linera.md:21]:
```suggestion
To interact with the Testnet, some tokens are needed. A faucet service is
```

File: src/developers/sdk/deploy.md (hunk 2 above)
Comment (afck) [reply src/developers/sdk/deploy.md:5]: I'd say neither "devnet" nor "testnet" nor "faucet" are proper names, so I'd lowercase them and use an article: "the testnet". (But I'm happy to do the opposite, too, as long as it's consistent.)

**Review by afck — APPROVED** [review]

**Pull request merged**